packages/sdk-aws-kms-signer/index.ts

```diff
@@ -0,0 +1,49 @@
+import { CredentialSigner, KeyAssertion } from '@dfns/sdk'
+import { toBase64Url } from '@dfns/sdk/utils'
+import { KMSClient, SignCommand, SigningAlgorithmSpec } from "@aws-sdk/client-kms";
+
+
+export class AwsKmsKeySigner implements CredentialSigner<KeyAssertion> {
+
+  constructor(
+    private options: {
+      kmsKeyId: string
```

Comment:
Should this one be optional ? If it takes the default `AWS_REGION` in environment ? Or maybe the other way around: should we rather allow people to add the whole kms config ? having `kmsConfig?: KMSClientConfig` in the params instead of `awsRegion`

Reply:
same train of thought as above, maybe create a sub type that groups the kms key parameters for the sign command together.

```
kmsKeyConfig: {
  id: string
  algorithm: SigningAlgorithmSpec
}
```

Reply:
👍  passing in the config is ideal, region is not required just copying from the docs 😅